### ISSUE TITLE
num_correspondences scale

### ISSUE THREAD

**Issue by wuqun-tju** [crockwell/far#3]
hello, I have one question , in th code , num_correspondences is divided by 500, 10 or 100， I want to know how to choose this constant scale?

**Comment by crockwell** [comment]
Good question! So there are 3 values for number of correspondences. (1) The number of inliers (2) the number of inliers at a tighter threshold (3) the number of inliers at an even tighter threshold. 

- We divide all 3 by 500 to make them of reasonable scale for the network ([here](https://github.com/crockwell/far/blob/main/mp3d_loftr/src/loftr/loftr.py#L164)) 
- (2) and (3) are inliers at thresholds of 1/10 and 1/100 the normal threshold, and are defined [here](https://github.com/crockwell/far/blob/main/mp3d_loftr/third_party/prior_ransac/ransac.py#L284). Their values aren't actually divided by 10 or 100, but rather their threshold is just different.

Hope that helps!
Chris

**Comment by wuqun-tju** [comment]
Thank you for your reply ! I still have confusions as follows:
 (1) the number 500 can be replace with another number？  I want to know how to chosse the number value
 (2) 10 and 100 is used [here](https://github.com/crockwell/far/blob/main/mapfree_6dreg/lib/models/regression/model.py#L211)

**Comment by crockwell** [comment]
(1) Sure, the number is arbitrary, picked to try to match ranges of other network outputs. I found this number worked well. 
(2) The reason I did this was a bit ad-hoc but I wanted the network to be trained with 3 input number of correspondences, even though only one number is available when using OpenCV RANSAC, which we use in early training of FAR (i.e. T1). 3 are available when using the full network, as the Kornia function is custom and can return three values. To get the network prepared for the range of values expected, I divided by 10 and 100 if only the OpenCV number of correspondences were available.
